aa_intercom/models.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.contrib.contenttypes import fields as generic
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.db.models.signals import post_save
+from django.utils.translation import ugettext_lazy as _
+
+from aa_intercom.signals import intercom_event_push_to_intercom_post_save
+
+
+class IntercomEvent(models.Model):
+    """
+        Fill types per project.
```

Comment:
This is not abstract so how we can fill it? Probably types should be in settings.